Description: Fix cross building from x86 to non-x86
 Caused by the dynamic recompiler selection confusing build and host
 architectures.
Author: Helmut Grohne <helmut@subdivi.de>
Bug-Debian: https://bugs.debian.org/900917
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -216,9 +216,9 @@ else
 fi
 
 if test "x$DYNARECSEL" = xauto; then
-	if expr x"$build_cpu" : 'xi.86' > /dev/null; then
+	if expr x"$host_cpu" : 'xi.86' > /dev/null; then
 		DYNARECSEL="x86"
-	elif expr x"$build_cpu" : 'xx86_64' > /dev/null; then
+	elif expr x"$host_cpu" : 'xx86_64' > /dev/null; then
 		DYNARECSEL="x86_64"
 	fi
 fi
